db/data.py

```diff
@@ -58,6 +58,44 @@ def _has_key(dictionary, key):
         dictionary = dictionary[part]
     return True
 
+def check_highlevel_meta():
+    with db.engine.begin() as connection:
+        query = text("""
+                        SELECT COUNT(*)
+                        FROM highlevel_meta ;
+                    """)
+        result = connection.execute(query).fetchall()
+    return result
+
+def clean_highlevel_data():
+    connection = db.engine.connect()
+    trans = connection.begin()
+    try:
+        query = text("""
+                    DELETE
+                    FROM highlevel
+                    WHERE highlevel.id in
+                        (   SELECT highlevel.id
+                            FROM highlevel
+                            EXCEPT
```

Comment:
I've never come across the select ... from ... except syntax before - that's not a problem, and it looks like this query works, but if we use left join in the count method, we can also use the same type of join here to do this query